app/aws-lsp-partiql-binary/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { PartiQLLanguageServer } from '@aws/aws-lsp-partiql'
```

Comment:
nit: `aws-` prefix in package name is redundant

Reply:
yeah that `aws` there is a bit redundant, removed it